schemas/options.json

```diff
@@ -0,0 +1,9 @@
+{
+  "type": "object",
+  "properties": {
+    "name": {
+      "type": "boolean"
+    }
+  },
+  "additionalProperties": false
+}
```

Comment:
_nit:_ Please add a blank line at the end of the file

Reply:
Will do

Reply:
Maybe just add `.editorconfig` file in the root directory of this source tree to avoid such issues in the future?
```sh
# EditorConfig helps developers define and maintain consistent
# coding styles between different editors and IDEs
# http://editorconfig.org

root = true

[*]

# Change these settings to your own preference
indent_style = space
indent_size = 2

# We recommend you to keep these unchanged
end_of_line = lf
charset = utf-8
trim_trailing_whitespace = true
insert_final_newline = true
```

Reply:
@frenzzy hm that file already exists at root. https://github.com/webpack-contrib/webpack-serve/blob/master/.editorconfig. I think the issue is that webpack-defaults writes files that don't get touched afterwards, and those files contain the bad style info.